<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LASH</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(160, 194, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Calcular resultados</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Número de dias (Máx. 5479)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="num_dias">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="calculate">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Calcular</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" rowspan="5">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>374</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>374</height>
       </size>
      </property>
      <property name="title">
       <string>Arquivos de entrada</string>
      </property>
      <widget class="QLineEdit" name="first_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Primeiro Arquivo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="first_ok">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Segundo Arquivo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="second_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="second_ok">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Terceiro Arquivo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="third_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="third_ok">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Procurar</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:32pt; font-weight:600; font-style:italic; color:#070042;&quot;&gt;LASH&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" rowspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Mostrar resultados para:</string>
      </property>
      <widget class="QCheckBox" name="checkBox_ETc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ETc</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_ETr">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ETr</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Vazao">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Vazão Total</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_At">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Armaz. Total</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Pts">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prec. Total</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Dcr">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ascensão capilar</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Ia">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Abst. Inicial</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_Pe">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prec. Efetiva</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_vESD">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Vazão_ESD</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_vESS">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>85</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Vazão_ESS</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_vEB">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>85</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Vazão_EB</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1" rowspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Parâmetros</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Kcr</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_kcr">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Kb</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_kb">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Kss</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_kss">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_css">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cb">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_coef">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cs</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Css</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cb</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Coef. Ia</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_parametros">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Usar valores padrão</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_maxn">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Maxn</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_peps">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ngs">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Peps</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>70</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ngs</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>&amp;Opções</string>
    </property>
    <addaction name="actionAjuda"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAjuda">
   <property name="text">
    <string>&amp;Ajuda</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>checkBox_ETc</tabstop>
  <tabstop>checkBox_ETr</tabstop>
  <tabstop>checkBox_Vazao</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
